src/macaron/repo_finder/repo_finder.py

```diff
@@ -113,8 +148,7 @@ def to_repo_path(purl: PackageURL, available_domains: list[str]) -> str | None:
     domain = to_domain_from_known_purl_types(purl.type) or (purl.type if purl.type in available_domains else None)
     if not domain:
         logger.info("The PURL type of %s is not valid as a repository type. Trying to find the repository...", purl)
-        # Try to find the repository
-        return find_repo(purl)
+        return None
```

Comment:
I wonder why we decided to return None here instead of `find_repo(purl)`

Reply:
It was redundant. The only callsite of this function performs the removed function call if None is returned.